ui/src/app/views/workflow/run/node/pipeline/workflow-run-job/workflow-run-job.component.ts

```diff
@@ -73,6 +70,10 @@ export class WorkflowRunJobComponent implements OnInit, OnDestroy {
 
     @ViewChild('jobVariable') jobVariable: WorkflowRunJobVariableComponent;
 
+    @Input() projectKey: string;
+    @Input() workflowName: string;
+    @Input() workflowRunNumber: number;
+
```

Comment:
All of this can be get in store instead of being pass as an input no?
It avoid some problem with async update of the input

Reply:
Context info will be moved to store when adding navigation between runs.